### PR TITLE
Add more details to the TypeScript docs

### PR DESCRIPTION
**Pull request by justinfagnani**
We hadn't mentioned what's required from a TypeScript config in this section yet.

### CODE REVIEW

**Review by augustjk — APPROVED** [review]
LGTM

File: packages/lit-dev-content/site/docs/v3/tools/development.md [hunk 1]
```diff
@@ -202,12 +202,23 @@ For full installation and usage instructions, see the [Web Dev Server documentat
 
 ## TypeScript { #typescript }
 
-TypeScript extends the Javascript language by adding support for types. Types are useful for catching errors early and making code more readable and understandable.
+Lit support developing components in TypeScript, including full type declarations for the Lit APIs, standard and experimental decorators, and community tools for template type-checking and linting.
+
+Because Lit is just a library, and doesn't require a compiler or use non-standard langauge syntax, there are no specific TypeScript tools that are required. Lit works with the official TypeScript compiler, `tsc`, with TypeScript wrappers such as those for Rollup, Vite, or Webpack, and alternate compilers like `esbuild`.
```
Comment (e111077) [inline packages/lit-dev-content/site/docs/v3/tools/development.md:207]: Clarify to reader what you’re implying by “just a library”

**Comment by e111077** [comment]
Oops it’s merged

**Comment by justinfagnani** [comment]
> Oops it’s merged

I can make those changes though!

**Pull request merged**